src/Features/Core/Portable/Completion/CommonCompletionProvider.cs

```diff
@@ -93,14 +95,12 @@ public override async Task<CompletionChange> GetChangeAsync(Document document, C
         protected virtual Task<TextChange?> GetTextChangeAsync(CompletionItem selectedItem, char? ch, CancellationToken cancellationToken)
             => SpecializedTasks.Default<TextChange?>();
 
-        private static readonly CompletionItemRules s_suggestionItemRules = CompletionItemRules.Create(enterKeyRule: EnterKeyRule.Never);
-
-        protected static CompletionItem CreateSuggestionModeItem(string displayText, string description)
+        protected static CompletionItem CreateSuggestionModeItem(string? displayText, string? description)
         {
             return CommonCompletionItem.Create(
                 displayText: displayText ?? string.Empty,
                 displayTextSuffix: "",
-                description: description != null ? description.ToSymbolDisplayParts() : default,
+                description: description == null ? default : description.ToSymbolDisplayParts(),
```

Comment:
```suggestion
                description: description?.ToSymbolDisplayParts() ?? default,
```

Reply:
thanks! i do prefer the ternary form though.  i really wish tehre was a `?.` form that gave you a `default(T)` not a `T?`.